dev/05_data_core.ipynb

```diff
@@ -1197,7 +1197,7 @@
        "\n",
        "> <code>Cuda.encodes</code>()\n",
        "\n",
-       "Return batch to CPU"
+       "Move batch to [`device`](https://pytorch.org/docs/stable/tensor_attributes.html#torch-device)"
```

Comment:
Just leave it as `device` without the link.

Reply:
The link, I figured out, is added automatically thanks to a call of
[_get_pytorch_index().get('device')](https://github.com/fastai/fastai_dev/blob/a5f2a60285240bfa7e8a6eb3bd19a66839fa45c5/dev/local/notebook/showdoc.py#L22).
Not sure yet how to eliminate. Please advice.

Reply:
Ah yes - apologies I misread it!